meta-aos-vm-main/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend

```diff
@@ -26,6 +26,10 @@ python do_update_config_append() {
         if node != d.getVar("NODE_ID"):
             data["RemoteIams"].append({"NodeID": node, "URL": node+":8089"})
 
+    if len(d.getVar("NODE_LIST").split()) == 1
```

Comment:
Do you see some problem with restarting NFS?

Reply:
alerts  on the cloud in case of single node.